raft-tech/TANF-app
# Feat/992 change file selection before submittion

## Summary of Changes
_Provide a brief summary of changes_
Pull request closes #992

https://user-images.githubusercontent.com/7119690/197221364-752c3376-564e-4536-87bd-aab87629e6ca.mov

https://user-images.githubusercontent.com/7119690/197221410-b0469a28-0b76-4873-9d08-ed39bdb442cb.mov

## How to Test
_List the steps to test the PR_

1.  Start app locally
2. Login and go to datafile upload page
3. Make your selection and click search
4. Change the selections and see the header change automatically

## Deliverables
_More details on how deliverables herein are assessed included [here](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverables)._

### [Deliverable 1: Accepted Features](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-1-Accepted-Features)

Checklist of ACs:
- [ ] Frontend raises [an error/warning notification](https://www.figma.com/file/irgQPLTrajxCXNiYBTEnMV/TDP-Mockups-For-Feedback?node-id=4203%3A22618) if user changes selections after upload but prior to submit
- [ ] Frontend matches mock ups

### [Deliverable 2: Tested Code](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-2-Tested-Code)

+ Are all areas of code introduced in this PR meaningfully tested?
+ Are code coverage minimums met?

### [Deliverable 3: Properly Styled Code](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-3-Properly-Styled-Code)


### [Deliverable 4: Accessible](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-4-Accessibility)



### [Deliverable 5: Deployed](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-5-Deployed)


### [Deliverable 6: Documented](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-6-Code-documentation)


### [Deliverable 7: Secure](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-7-Secure)


### [Deliverable 8: User Research](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-8-User-Research)

Research product(s) clearly articulate(s):
